ark/utils/load_utils.py

```diff
@@ -479,3 +483,92 @@ def load_tiled_img_data(data_dir, fovs, expected_fovs, channel, single_dir, file
                           dims=["fovs", "rows", "cols", "channels"])
 
     return img_xr
+
+
+def fov_to_ome(data_dir: Union[str, pathlib.Path], ome_save_dir: Union[str, pathlib.Path],
```

Comment:
You should talk to @cliu72 about where she wants this in the repo. I'm guessing a utility notebook that explains how to convert a folder of OMEs to a folder of directories, and vice versa, but up to her.

Reply:
I think utility notebooks would be easier, what do you think @srivarra? Timeline that we would need this done for my resubmission is probably ~1-1.5 months. If you guys want to have a flag in all the import functions instead, that's obviously also fine, but not sure how long that would take.

Reply:
@cliu72 I would agree, a small utility notebook which converts a directory of OME-TIFFs and moves them to `data/your_dataset` would be straightforward.